brand new story

```diff
@@ -0,0 +1 @@
+It was 5 am in the morning. John Doe was looking at his steaming cup of coffee and was thinking about the previous 72 hours of his life.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.